ang/crmUi.js

```diff
@@ -642,7 +642,7 @@
             $timeout(function () {
               // ex: msg_template_id adds new item then selects it; use $timeout to ensure that
               // new item is added before selection is made
-              element.select2('val', ngModel.$viewValue);
+              element.select2('val', ngModel.$modelValue);
```

Comment:
@TobiasLounsbury, can you please complete the changes discussed above and also make a PR into 4.6? Thanks!